odl/tomo/geometry/parallel.py

```diff
@@ -308,7 +308,7 @@ def __init__(self, angle_intvl, dparams, agrid=None, dgrid=None,
                 raise ValueError('length ({}) of axis {} is not 3'.format(
                     len(axis), axis))
         self._axis = np.array(axis) / np.linalg.norm(axis)
-        self._detector = Flat2dDetector(dparams, dgrid)
+        self._detector = Flat2dDetector(dparams, [[0, 1, 0], [0, 0, 1]], dgrid)
```

Comment:
That may very well end up as the only place we use this internally. Nothing bad about this, I guess, if it offers flexibility to the user.